lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -461,7 +461,12 @@ defmodule ExUnit.Formatter do
 
   defp format_message(value, formatter) do
     value = String.replace(value, "\n", "\n" <> @counter_padding)
-    formatter.(:error_info, value)
+
+    if String.contains?(value, IO.ANSI.red()) do
```

Comment:
Instead of checking for red specifically, should we check for any color? Or perhaps the reset?

Reply:
I think all of them will have the reset in common, so I think it's simpler and faster to check for it